lib/template/pages/schedule.jade

```diff
@@ -15,7 +15,7 @@
         =' '
         | Lee White Show "Your 15 Minutes of Fame",
         =' '
-        a(href=$ren->url('artists').'#theCrumbs') The Crumbs, CA
+        a(href=$ren->url('artists').'#theCrumbs') The Crumbs, CAN
```

Comment:
Mohl bys to upravit tady? Bude to rychlejší, protože nemám přístupová práva k tvému repozitáři.